docs/ui/Dont.tsx

```diff
@@ -0,0 +1,30 @@
+import { Text } from '@marigold/components';
```

Comment:
I think we can put them in one file, makes it easier to find them and we need to adjust both anyway when we need to change something.